antarest/study/storage/variantstudy/model/command/icommand.py

```diff
@@ -102,5 +108,15 @@ def _create_diff(self, other: "ICommand") -> List["ICommand"]:
     def get_inner_matrices(self) -> List[str]:
         raise NotImplementedError()
 
+    def _get_command_extraction(self) -> "CommandExtraction":
```

Comment:
_get_command_extractor (you can rename also the CommandExtraction class if you want)